### PR TITLE
new icon: C3js (original, plain)

### PR DESCRIPTION
**Pull request by mjlabe**
C3.js icon from PR #136

c3js-original

c3js-plain

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -111,6 +111,14 @@
       "font": ["plain", "plain-wordmark", "line", "line-wordmark"]
     }
   },
+  {
+    "name": "c3js",
+    "tags": [],
```
Comment (amacado) [inline devicon.json:116]:
```suggestion
    "tags": ["javascript","chart","library"],
```

**Review by amacado — CHANGES_REQUESTED** [review]
Hello @mjlabe and thanks for your contribution. Looks fine, I would only like to improve the tagging.

**Comment by mjlabe** [comment]
Done! @amacado

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Thomas-Boi** [comment]
Hi @mjlabe,

I'm working on merging your PR to accept your icon. However, it seems like your branch is a bit out of date. It'd be much appreciated if you can bring the branch up to date with develop and re-push.

After a week, if the branch hasn't been updated I'll make a new PR cherry-picking your commit. This is to ensure you are still the author of the commit and get proper credits. You can also just give me permission to do it this way now.

It'd be quicker if you update the branch yourself though but whichever works 👍.

You got quite a few icons 😄

**Comment by Thomas-Boi** [comment]
Hi @mjlabe, I am unable to find the the official C3.js logo that matches the one you provided. In fact, it looks like your C3.js logo is similar to the D3.js logo but with the D flipped. Here's the [logo](https://avatars1.githubusercontent.com/u/9966497?s=280&v=4) that I found, which was from [this page](https://c3js.org/)

If you can provide any proof that your logo is the correct one, we'll accept your icon. Until then, we will have to close this PR.

**Pull request closed without merging**